src/Veldrid/RgbaByte.cs

```diff
@@ -123,5 +136,75 @@ public override string ToString()
         {
             return string.Format("R:{0}, G:{1}, B:{2}, A:{3}", R, G, B, A);
         }
+
+        /// <summary>
+        /// Multiplies the channels of an RgbaByte by the given value.
+        /// </summary>
+        /// <param name="value">The source RgbaByte.</param>
+        /// <param name="scale">The value to multiply by.</param>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static RgbaByte Multiply(RgbaByte value, float scale)
+        {
+            return new RgbaByte(
```

Comment:
This has bad behavior in cases where channels overflow. RgbaFloat has slightly better semantics since it uses floats, but at least that domain is better understood for representing colors. If you overflow an RgbaByte, your color is basically meaningless. If you clamp to (255, 255, 255, 255), then the scaling becomes skewed. I'm not really sure how to handle this other than to remove the stuff from RgbaByte...

Reply:
MonoGame uses clamp apparently...